### ISSUE TITLE
Add ecosyste.ms to guide

### ISSUE THREAD

**Issue by benmccann** [43081j/ecosystem-cleanup#55]
If you find a bloated package, you can find the most downloaded packages that contain it as a dependency to target for cleanup. It's a really great site. It's still a little buggy, so not sure if we should list it yet, but we should use and test it internally and file issues and then list it after it's stable. The maintainer is super responsive

https://packages.ecosyste.ms/registries/npmjs.org/packages/deep-equal/dependent_packages?kind=runtime&latest=true&order=desc&sort=downloads

**Comment by 43081j** [comment]
makes sense to me. if it turns out to be useful enough im open to us adding it to the guide for sure

maybe lets revisit some time soon and add it if we've been making good use of it

**Comment by AbhiPrasad** [comment]
I've been finding it really useful to dig into the packages talked about it in https://github.com/es-tooling/module-replacements and learning how they are being used.

I kinda wish there was a weighted mode though, where I could see packages that were recently updated + had a high amount of dependents. These are the highest ROI in my opinion. Right now you have to individually filter for recently updated and cross reference it with download count/dependent count.

**Comment by 43081j** [comment]
that's true, quite a lot of packages we _could_ contribute to are just no longer active really (but still heavily depended on)

so in those cases its probably better to spend our time contributing to alternatives or packages which are still maintained
